src/features/content-explorer/content-explorer/ContentExplorerBreadcrumbs.js

```diff
@@ -28,23 +28,25 @@ const ContentExplorerBreadcrumbs = ({
         >
             <IconChevron direction="left" size="6px" color="#333" />
         </Button>
-        <Breadcrumb label={formatMessage(messages.breadcrumb)}>
+        <Breadcrumb label={formatMessage(messages.breadcrumb)} {...breadcrumbProps}>
             {foldersPath.map((folder, i) => (
-                <PlainButton
+                <div
                     className="lnk"
                     key={folder.id}
-                    title={folder.name}
                     onClick={event => onBreadcrumbClick(i, event)}
+                    role="presentation"
```

Comment:
If we are updating the above to a `div` the aria role might have to be set to `button`.